docs/_archive/debt/debt-357-test-double-discipline-drift.md

```
@@ -2,8 +2,13 @@
 
 **Priority:** P3
 **Created:** 2026-04-08
+**Status:** Resolved (PR #273, merged 2026-04-10)
 **Source:** Follow-up from [DEBT-354](./debt-354-god-file-and-clean-code-audit.md)
-**Related:** [docs/dev/react-vitest-testing.md](../dev/react-vitest-testing.md), [clerk-auth-gateway.test.ts](../../src/adapters/gateways/clerk-auth-gateway.test.ts), [get-started-cta.test.tsx](../../components/get-started-cta.test.tsx), [auth-nav.test.tsx](../../components/auth-nav.test.tsx)
+**Related:** [docs/dev/react-vitest-testing.md](../../dev/react-vitest-testing.md), [clerk-auth-gateway.test.ts](../../src/adapters/gateways/clerk-auth-gateway.test.ts), [get-started-cta.test.tsx](../../components/get-started-cta.test.tsx), [auth-nav.test.tsx](../../components/auth-nav.test.tsx)
+
+### Resolution
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)